<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeRemap_UI</class>
 <widget class="QWidget" name="TimeRemap_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="6" column="0">
    <widget class="QCheckBox" name="pitch_compensate">
     <property name="text">
      <string>Pitch compensation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QToolButton" name="button_del">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="edit-delete">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QCheckBox" name="output_seek">
     <property name="text">
      <string>Seek monitor</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QCheckBox" name="source_seek">
     <property name="text">
      <string>Seek monitor</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>28</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="button_prev">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="go-previous">
       <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QFrame" name="info_frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Source time</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="inLayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Output time</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="outLayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Speed before</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="speedBefore">
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>After</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="speedAfter">
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QVBoxLayout" name="remapLayout"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Source clip</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="QCheckBox" name="frame_blending">
     <property name="text">
      <string>Frame blending</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="button_next">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="go-next">
       <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="button_add">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QCheckBox" name="move_next">
     <property name="text">
      <string>Preserve speed of next keyframes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
